### PR TITLE
Rename PipelineShaderOptions noContract to noContractOpDot.

### PR DESCRIPTION
**Pull request by perlfu**
It only effects OpDot in SPIRV reader.

### CODE REVIEW

**Comment by amdvlk-admin** [comment]
### Test summary for commit a95089a829e4a90bad21ff13bdb036579ce5f443
<details><summary>CTS tests (Failed: 0/209351)</summary><ul><li>Built with version 1.3.3.1</li>
<details open><summary>Rhel 9.0, Gfx10</summary><ul>

<li>Passed: 34735/69788 (49.8%)</li>
<li>Failed: 0/69788 (0.0%)</li>
<li>Not Supported: 35053/69788 (50.2%)</li><li>Warnings: 0/69788 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 22.04, Navi3x</summary><ul>

<li>Passed: 38620/69788 (55.3%)</li>
<li>Failed: 0/69788 (0.0%)</li>
<li>Not Supported: 31168/69788 (44.7%)</li><li>Warnings: 0/69788 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Navi2x</summary><ul>

<li>Passed: 38654/69775 (55.4%)</li>
<li>Failed: 0/69775 (0.0%)</li>
<li>Not Supported: 31121/69775 (44.6%)</li><li>Warnings: 0/69775 (0.0%)</li>
</ul></details>


</ul></details>

File: include/vkgcDefs.h [hunk 1]
```diff
@@ -79,6 +79,7 @@
 //  %Version History
 //  | %Version | Change Description                                                                                    |
 //  | -------- | ----------------------------------------------------------------------------------------------------- |
+//  |     65.2 | Rename noContract in PipelineShaderOptions to noContractOpDot
```
Comment (nhaehnle) [inline include/vkgcDefs.h:82]: This is technically not a backwards-compatible change, is it? Let's bump the major version.

Comment (perlfu) [reply include/vkgcDefs.h]: Done. There should be no breakage either way as this is only used internally within the compiler.

**Review by nhaehnle — APPROVED** [review]
Yeah. Still, thanks!

**Comment by perlfu** [comment]
Jenkins failure had nothing to do with this commit.
Rebase and force push (no changes), to retrigger testing.

**Comment by amdvlk-admin** [comment]
### Test summary for commit 5b589e5e9de233694e84b89b653cab89f9898459
<details><summary>CTS tests (Failed: 0/209351)</summary><ul><li>Built with version 1.3.3.1</li>
<details open><summary>Rhel 9.0, Gfx10</summary><ul>

<li>Passed: 34735/69788 (49.8%)</li>
<li>Failed: 0/69788 (0.0%)</li>
<li>Not Supported: 35053/69788 (50.2%)</li><li>Warnings: 0/69788 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 22.04, Navi3x</summary><ul>

<li>Passed: 38620/69788 (55.3%)</li>
<li>Failed: 0/69788 (0.0%)</li>
<li>Not Supported: 31168/69788 (44.7%)</li><li>Warnings: 0/69788 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Navi2x</summary><ul>

<li>Passed: 38654/69775 (55.4%)</li>
<li>Failed: 0/69775 (0.0%)</li>
<li>Not Supported: 31121/69775 (44.6%)</li><li>Warnings: 0/69775 (0.0%)</li>
</ul></details>


</ul></details>

**Pull request merged**